packages/palette/src/elements/Input/Input.tsx

```diff
@@ -105,14 +112,14 @@ const StyledInput = styled.input<StyledInputProps>`
   appearance: none;
   line-height: 1;
   border: 0;
-  border-bottom: 1px solid;
-  border-radius: 0;
+  border: 1px solid ${themeGet("colors.black15")};
+  border-radius: 3px;
   transition: border-color 0.25s, color 0.25s;
   font-family: ${themeGet("fonts.sans")};
   ${systemHeight};
 
   ::placeholder {
-    transition: color 0.25s;
+    transition: color 0.25s, opacity 0.2s;
```

Comment:
Is there a spec for the animation? As far as transition values go, etc.

Reply:
I don't think there are any specific specs for the animation speed, so I'll change it to be .25 to match the current speed we have 👍